static/app/data/platformCategories.tsx

```diff
@@ -409,6 +409,7 @@ const customMetricBackendPlatforms: readonly PlatformKey[] = [
 ];
 
 const customMetricFrontendPlatforms: readonly PlatformKey[] = [
```

Comment:
```suggestion
const customMetricPlatforms: readonly PlatformKey[] = [
```
now that it includes also backend frameworks?

Reply:
or even better, I could use the existing`customMetricBackendPlatforms` set 😬